ng-dev/pr/common/validation/validation-config.ts

```diff
@@ -21,6 +21,8 @@ export class PullRequestValidationConfig {
   assertCompletedReviews = true;
   assertEnforcedStatuses = true;
   assertMinimumReviews = true;
+  assertIsolatePrimitives = true;
```

Comment:
These shouldn't default to false, because they shouldn't run in every repository.